src/controller/python/chip/enum.py

```diff
@@ -0,0 +1,49 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+from aenum import IntEnum as AenumsIntEnum
+from aenum import extend_enum
+
+_add_missing_enum_value_as_is = False
+_map_missing_enum_to_unknown_enum_value = True
+_dummy_count = 0
+
+
+class IntEnum(AenumsIntEnum):
```

Comment:
Why is this using the same name as the standard IntEnum? The generated code would need to use this one anyway.

Suggest calling it `MatterIntEnum` or such